fail2ban/server/failmanager.py

```diff
@@ -43,6 +44,33 @@ def __init__(self):
 		self.__maxRetry = 3
 		self.__maxTime = 600
 		self.__failTotal = 0
+		self.__subnetMask = None
+		
+	def setSubnetMask(self, value):
+		try:
+			self.__lock.acquire()
+			if "." in value: # 255.255.255.0 style mask
+				maskparts = value.split(".")
+				if len(maskparts) != 4:
+					raise ValueError("subnetmask must be valid")
```

Comment:
I prefer to give as informative error message as possible, e.g. here  `"subnetmask %s is not valid since doesn't conform 255.255.255.255 style mask" % value`